File: go-client-sdk/pkg/models/shared/taskupdate.go
package shared

type TaskUpdate struct {
	Additional map[string]interface{} `json:"additional,omitempty"`
	Body       *string                `json:"body,omitempty"`
	DueDate    *string                `json:"dueDate,omitempty"`
	IsDone     *bool                  `json:"isDone,omitempty"`
	Priority   *string                `json:"priority,omitempty"`
	Status     *string                `json:"status,omitempty"`
	Subject    *string                `json:"subject,omitempty"`
	UserID     *string                `json:"userId,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/taskupdate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class TaskUpdate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=additional" })
  additional?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=body" })
  body?: string;

  @SpeakeasyMetadata({ data: "json, name=dueDate" })
  dueDate?: string;

  @SpeakeasyMetadata({ data: "json, name=isDone" })
  isDone?: boolean;

  @SpeakeasyMetadata({ data: "json, name=priority" })
  priority?: string;

  @SpeakeasyMetadata({ data: "json, name=status" })
  status?: string;

  @SpeakeasyMetadata({ data: "json, name=subject" })
  subject?: string;

  @SpeakeasyMetadata({ data: "json, name=userId" })
  userId?: string;
}
